backend/server/api.go

```diff
@@ -51,6 +51,13 @@ func bindAPIRoutes(r *mux.Router, db *database.Connection, contentStore contents
 		return services.ListOperations(r.Context(), db)
 	}))
 
+	route(r, "GET", "/api/checkconnection", jsonHandler(func(r *http.Request) (interface{}, error) {
+		reply := struct {
+			Connected bool `json:"ashirtConnected"`
```

Comment:
Can we change this to 
```go
Ok bool `json:"ok"`
```

Reply:
We can. my motivation here was to make sure we were connecting to an ashirt server, but this kind of issue seems rare anyway. Parsing Ok only should be fine.